pkg/agent/sysadvisor/plugin/poweraware/advisor/action/strategy/strategy.go

```diff
@@ -57,7 +57,10 @@ func (p ruleBasedPowerStrategy) RecommendAction(actualWatt int,
 
 	if ttl <= time.Minute*2 {
 		// whatever valid alert, power capping should do in short of 2 minutes
-		return action.PowerAction{Op: spec.InternalOpFreqCap, Arg: desiredWatt}
+		// this the default behavior unless instructed explicitly by internal op
```

Comment:
Should this be part of the `autoAction` function?

Reply:
yes it is actually. will update code NOT to return directly here, and let autoAction to fill up the needed detail.